### ISSUE TITLE
Fix input-group reset border radius

### ISSUE THREAD

**Issue by florianlacreuse** [angel-vladov/select2-bootstrap-theme#10]
Hi @angel-vladov,

I found an issue with the reset border radius rules.

In BS4, `.form-control` can be inside an `.input-group` without prepend or append elements. In this case the `.form-control` keeps its default borders radius.

With Select2 components, the right borders radius will be reset to 0, like this:

![2020-01-29_11-35](https://user-images.githubusercontent.com/10129873/73349434-7770f400-428b-11ea-9eb0-f7f9ff045435.png)

The current scss rules:

https://github.com/angel-vladov/select2-bootstrap-theme/blob/8de55f55e3db484b1213d488ea3b7f756187a3af/src/select2-bootstrap.scss#L810-L825

IMHO this could be rewrite as follows (easier and fix the issue):

```scss
.input-group > .select2-hidden-accessible {
  &:not(:first-child) + .select2-container--bootstrap > .selection > .select2-selection
  &:not(:first-child) + .select2-container--bootstrap > .selection > .select2-selection.form-control {
     @include border-left-radius(0);
  }

  + .select2-container--bootstrap:not(:last-child) > .selection > .select2-selection,
  + .select2-container--bootstrap:not(:last-child) > .selection > .select2-selection.form-control {
    @include border-right-radius(0);
  }
}
```

**Comment by angel-vladov** [comment]
Thanks for catching that. Will push an update today

**Comment by angel-vladov** [comment]
There's a new release in npm. Version is changed to `0.2.0-beta.5` and includes your suggestion.
_Thanks again for catching this_

**Comment by florianlacreuse** [comment]
Thanks for your time and effort!

This BS theme is important for our applications, so I'll let you know if we find another issue in the future. Keep up the good work!

**Comment by florianlacreuse** [comment]
@angel-vladov I'm sorry, my bad, I forgot a comma when typing the rules for the left side in my first message...

Here is the missing comma in the current scss file: 

https://github.com/angel-vladov/select2-bootstrap-theme/blob/1c29d9c45cfa9449ef4ebd56b4a79987b2a37adb/src/select2-bootstrap.scss#L811

**Comment by angel-vladov** [comment]
Good catch. I missed it as well. Published new version `0.2.0-beta.6` that has the missing comma
